### DISCUSSION TITLE
某个数据可以单独设置过期时间么?

### DISCUSSION THREAD

**Discussion by zf2187** [orca-zhang/ecache#14] [Q&A] [answered]
如果不用LRU-2,某些经常获取的数据,我希望可以单独延长过期时间,是否可以增加这样的接口?

**Comment by orca-zhang** [comment, marked as answer]
可以参考常见问题2噢，热点数据可以分开存储，也可以用`Inspect`方法DIY把两个或多个实例关联起来，做成冷热分层
